app/views/layouts/application.html.haml

```diff
@@ -25,8 +25,9 @@
     %a{:href => '#content'} Skip to Content
     |
     %a{:href => '#nav'} Skip to Page Navigation
+  - if params[:controller] != 'vocab_sheets'
+    = render :partial => 'shared/test_banner'
```

Comment:
can we follow rubocop? I'm not sure why it's not flagging this, but i think it should be
```= render :partial => 'shared/test_banner' unless params[:controller] == 'vocab_sheets'``

Reply:
aslo, the new ruby has syntax

Reply:
also :elephant: and you are awesome!